app/views/students/_in_session.html.erb

```diff
@@ -9,10 +9,16 @@
         <% else %>
           <%= link_to 'Sign out', sign_out_path, id: 'attendance-sign-out-link' %><br>
           <% if current_student.pairs_on_day(Time.zone.now.to_date).any? %>
-            You are signed in with <%= current_student.pairs_on_day(Time.zone.now.to_date).map {|s| s.name}.join(' & ') %>. [<%= link_to 'change pair', sign_in_path, remote: true, id: 'attendance-change-pair-link' %>]
+            <span id='signed-in-with'>You are signed in with <%= current_student.pairs_on_day(Time.zone.now.to_date).map {|s| s.name}.join(' & ') %>. [<%= link_to 'change pair', sign_in_path, remote: true, id: 'attendance-change-pair-link' %>]</p>
           <% else %>
             <strong class='text-primary'>You are signed in without a pair!</strong> Click <%= link_to 'here', sign_in_path, remote: true, id: 'attendance-change-pair-link' %> to log your pair.
           <% end %>
+          <% if current_student.inverse_orphan_pairs_on_day(Time.zone.now.to_date).any? %>
+            <p class='alert alert-danger'>
+              <strong>Additional students have marked you as a pair today: <%= current_student.inverse_orphan_pairs_on_day(Time.zone.now.to_date).order(:name).pluck(:name).join(' & ') %></strong><br>
```

Comment:
This is too much logic for a view. Call `current_student.inverse_orphan_pairs_today.pluck(:name).join(' & ')`.

Reply:
Do you mean just make an additional method inverse_orphan_pairs_today that just calls inverse_orphan_pairs_on_day(Time.zone.now.to_date)?

Reply:
Yeah exactly. I dunno, maybe it's not necessary, just feels like a lot of Ruby for a view.

Reply:
Agreed. I keep considering just defaulting day to Time.zone.now.to_date on all those methods, so if calling @student.pairs_on_day it will auto-default to today if not passed a date. Does that make sense, or do you think it's better to write separate methods called pairs_today, etc, for clarity?

Reply:
I would lean towards more clarity.